src/libraries/Directory.Build.props

```diff
@@ -7,16 +7,12 @@
   <PropertyGroup>
     <ToolSetCommonDirectory>$(RepoRoot)artifacts\toolset\Common\</ToolSetCommonDirectory>
     <IsSourceProject>$([System.Text.RegularExpressions.Regex]::IsMatch($(MSBuildProjectDirectory), 'src%24'))</IsSourceProject>
+    <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.depproj'">$(RepositoryEngineeringDir)depProj.common.targets</LanguageTargets>
+    <RuntimePropsFile>$(RepoRoot)artifacts\tmp\$(Configuration)\RuntimeOS.props</RuntimePropsFile>
```

Comment:
Instead use `ArtifactsDir`? 

Also, we should be using `$([MSBuild]::NormalizeDirectory())` and `$([MSBuild]::NormalizePath())` functions for all these new path evaluations we introduce.